services/blockchain-indexer/shared/utils/regex.js

```diff
@@ -14,7 +14,9 @@
  *
  */
 const TOKEN_ID = /[0-9A-Fa-f]{16}/g;
```

Comment:
We can ignore the global flag in our regexes since all the inputs are to be exactly matched just once.
```suggestion
const TOKEN_ID = /[0-9A-Fa-f]{16}/;
```